app/CocoaPods/Base.lproj/MainMenu.xib

```diff
@@ -348,11 +354,68 @@
+                        <items>
+                            <menuItem title="Comment Selection" keyEquivalent="/" id="jCB-46-dI7">
+                                <connections>
+                                    <action selector="commentSelection:" target="-1" id="JLH-y1-1Rg"/>
+                                </connections>
+                            </menuItem>
+                        </items>
+                    </menu>
+                </menuItem>
+                <menuItem title="View" id="zmc-J6-DtQ">
+                    <modifierMask key="keyEquivalentModifierMask"/>
+                    <menu key="submenu" title="View" id="5gb-ap-wV7">
+                        <items>
+                            <menuItem title="Podfile Editor" keyEquivalent="1" id="Vad-pU-joO">
                                 <connections>
-                                    <action selector="commentSelection:" target="-1" id="RPd-Au-yfT"/>
+                                    <action selector="showEditorTab:" target="-1" id="s5O-hb-kgc"/>
                                 </connections>
                             </menuItem>
+                            <menuItem title="Console" keyEquivalent="2" id="N6r-GM-4n1">
+                                <connections>
+                                    <action selector="showConsoleTab:" target="-1" id="34F-FY-efO"/>
+                                </connections>
+                            </menuItem>
+                            <menuItem title="Information" keyEquivalent="3" id="2aY-CM-dhd">
+                                <connections>
+                                    <action selector="showInformationTab:" target="-1" id="Ds2-5n-OQs"/>
+                                </connections>
+                            </menuItem>
+                            <menuItem isSeparatorItem="YES" id="cEG-Nk-FNM"/>
+                        </items>
+                    </menu>
+                </menuItem>
+                <menuItem title="Build" id="uYo-Yr-gGi">
+                    <modifierMask key="keyEquivalentModifierMask"/>
+                    <menu key="submenu" title="Build" id="jb8-1H-tfs">
+                        <items>
+                            <menuItem title="Install" keyEquivalent="b" id="2QS-4L-qhV">
+                                <connections>
+                                    <action selector="install:" target="-1" id="V5Z-cM-DCd"/>
+                                </connections>
+                            </menuItem>
+                            <menuItem title="Install (Verbose)" alternate="YES" keyEquivalent="b" id="k0r-Kk-Xqp">
+                                <modifierMask key="keyEquivalentModifierMask" option="YES" command="YES"/>
+                                <connections>
+                                    <action selector="installVerbose:" target="-1" id="75H-69-ywR"/>
+                                </connections>
+                            </menuItem>
```

Comment:
This `alternate="YES"` thing is nice, you put two menu items next to each other, and pressing alt switches between them.

![screen shot 2015-12-13 at 1 14 56 pm](https://cloud.githubusercontent.com/assets/49038/11768466/857ef104-a19b-11e5-8716-9d436317af7b.png)